reactor-core/src/test/java/reactor/core/publisher/SinksTest.java

```diff
@@ -165,6 +165,39 @@ void noReplayBeforeFirstSubscriberIfEarlyError() {
 		}
 	}
 
+	@Nested
+	class MulticastReplayDuration {
+
+		final Duration            duration = Duration.ofMillis(1000);
+		final Sinks.Many<Integer> replaySink = Sinks.many().replay().limit(duration);
```

Comment:
please at least setup the `replaySink` in a `@BeforeEach` so that each test get their own

Reply:
right, will fix it